.github/workflows/ci.yml

```diff
@@ -0,0 +1,30 @@
+on:
+    pull_request: 
+    push:
+        branches: 
+            - main
+
+env:
+    NUSHELL_CARGO_PROFILE: ci
+    NU_LOG_LEVEL: DEBUG
+
+jobs:
+    my-job:
+        runs-on: ubuntu-latest
+
+        steps:
+            - uses: actions/checkout@v4
+
+            - uses: hustcer/setup-nu@v3.9
+              with:
+                version: '*'
+                check-latest: true
+                  # version: "0.80" # Don't use 0.80 here, as it was a float number and will be convert to 0.8, you can use v0.80/0.80.0 or '0.80'
```

Comment:
that might not be useful to keep?

Reply:
Yeah, it's not a useful comment. To test the CI in my machine I use [`act`](https://github.com/nektos/act) and for some reason if I `version: '*'` it grabs nushell v0.88 🤷🏼 . So in my pc I just comment out version and uncomment the line here 👍🏼 .
I'm deleting!